bin/eshost.js

```diff
@@ -63,11 +63,18 @@ const yargv = yargs
   .nargs('delete', 1)
   .describe('args', 'set arguments for a host entry (use with --add)')
   .nargs('args', 1)
+  .describe('configure-jsvu', 'Configure JSVU hosts in the config')
+  .boolean('configure-jsvu')
+  .describe('jsvu-root', 'Use this path containing the .jsvu folder (use this option if .jsvu is located somewhere other than the home directory).')
```

Comment:
Hmm, why would it be? `~/.jsvu` is hardcoded in jsvu (at least for now). Is this option there just in case this becomes configurable in the future?

Reply:
I suppose so -- I didn't realize it wasn't configurable :P

Reply:
I think I'll keep it in case it becomes supported in the future.